executor/simple.go

```diff
@@ -586,7 +586,8 @@ func (e *SimpleExec) executeBegin(ctx context.Context, s *ast.BeginStmt) error {
 	// With START TRANSACTION, autocommit remains disabled until you end
 	// the transaction with COMMIT or ROLLBACK. The autocommit mode then
 	// reverts to its previous state.
-	e.ctx.GetSessionVars().SetStatusFlag(mysql.ServerStatusInTrans, true)
+	e.ctx.GetSessionVars().SetInTxn(true)
+	txnCtx.IsExplicit = true
```

Comment:
why do the assignment again?

Reply:
Sorry, I forgot to remove it during refactoring